TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -463,10 +479,16 @@ function Execute-DacpacDeployment
     )
 
     Write-Verbose "Entering script SqlPackageOnTargetMachines.ps1"
+
+    Write-Verbose "Finding files with pattern $dacpacFile"
+    $dacpacFiles = Find-VstsFiles -LegacyPattern "$dacpacFile"
```

Comment:
Isn't Find-VstsFiles coming from task lib? How will this be available in case of WinRM remote execution of this script?